ReSharper.FSharp/src/FSharp.Psi.Services/src/Generate/GenerateSignatureProvider.fs

```diff
@@ -146,6 +146,29 @@ type FSharpGenerateSignatureBuilder() =
                 processModuleLikeDeclaration (indentation + moduleDecl.GetIndentSize()) nestedNestedModule nestedSigModule
             | :? IOpenStatement as openStatement ->
                 openStatement.Copy()
+            | :? ILetBindingsDeclaration as letBindingsDeclaration ->
+                
+                let sourceString (binding: IBinding) =
+                    let sb = StringBuilder()
+
+                    sb.Append("val ") |> ignore
+                    sb.Append(binding.HeadPattern.GetText()) |> ignore
+                    sb.Append(": ") |> ignore
+                    
+                    let refPat = binding.HeadPattern.As<IReferencePat>()
+                    if isNotNull refPat then
+                        let symbolUse = refPat.GetFcsSymbolUse()
+                        if isNotNull symbolUse then
+                            let mfv = symbolUse.Symbol :?> FSharpMemberOrFunctionOrValue
+                            sb.Append(mfv.FullType.Format(symbolUse.DisplayContext)) |> ignore
+                  
+                    sb.ToString()
+
+                let sigStrings =
+                    Seq.map sourceString letBindingsDeclaration.Bindings
+                    |> String.concat System.Environment.NewLine
```

Comment:
I think we can just use `"\n"` in this case, as Rider internally uses `lf` as the line ending.
Does this matter @auduchinok?

Reply:
I think it doesn't matter much, at least for now. I don't know the preferred way when creating new files, but I'd start with reusing new lines from the existing file, i.e. just use `lineEnding` from line 62.

We may also start supporting this plugin in ReSharper in VS future, and I don't think that they have this "always `"\n"`" logic there.